vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -181,7 +203,7 @@ func (r RefCountsMap) Init(d drivers.VolumeDriver, mountDir string, name string)
 	err = r.discoverAndSync(c, d)
```

Comment:
yes, good point. I was thinking "in the worst case, the mount request picks up the earlier data, which is not different from the recalc() simply hitting a few seconds later" but I forgot that the mount does modify the entry.
Lock is needed, per entry or per map. Of course the problem is - if vDVS waits on this lock it cannot reply to Docker. Docker (at least in the past) would also lock internally on the mount table, we got a deadlock and back to square 1..

Maybe we should just fail mount/unmount requests when recalc is in flight ?